README.md

```diff
@@ -14,6 +15,42 @@ Just add this to your `Cargo.toml`:
 hibitset = "0.6"
 ```
 
+## Using the `serde` feature
+
+There is an optional feature to use `serde` for serialization/deserializtion. To enable this feature add the following to your `Cargo.toml`:
+
+```
+[dependencies]
+hibitset = { version = "0.6.3", features = ["serde"]}
```

Comment:
```suggestion
hibitset = { version = "0.6.3", features = ["serde"] }
```

Reply:
Great, let me know if you need anything else.